### ISSUE TITLE
Add setting to hide downvoted posts

### ISSUE THREAD

**Issue by ErikHumphrey** [danbooru/danbooru#5643]
Add an advanced setting to hide downvoted posts instead of having to add `-downvote:albert` to each search.

This way, the user will never see posts that they've downvoted again, even when searching normally. If an opinionated user votes on every post, they'll only ever see the good ones after they're done downvoting.

Additionally or alternatively, the blacklist feature doesn't support blacklisting downvoted posts by username or user ID. Some might prefer that.

**Comment by MiiaBestLamia** [comment]
Agreed. I utilize the userscript that has this functionality, and would love to see it added to the base website. Loads of posts that I absolutely do not want to see bypass my blacklist, so getting rid of them via downvote is the only option.

On the topic of downvotes - I believe they currently serve no real purpose on danbooru.
Post score has no correlation with the quality of the artwork, most often a high score just indicates that a post is explicit, a low score indicates that a post has an unpopular fetish (bara comes to mind as an example). Poor quality posts' downvotes don't matter, as those get flagged and deleted.

I'd propose hiding downvotes from posts, so neither uploaders nor users see the amount of downvotes on their posts (only upvotes and favorites remain), but keeping the downvotes in the database, adding the aforementioned option to hide downvoted posts.
